sql/2022/security/tls_ca_issuers_pages.sql

```diff
@@ -0,0 +1,29 @@
+#standardSQL
+#Distribution of CA issuers for all pages
+SELECT
+  client,
+  issuer,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total_https_pages,
```

Comment:
Is this the # of pages or # of requests?

Reply:
This should be the number of pages, since only requests are selected where `NET.HOST(page)` equals `NET.HOST(url)`:
```SQL
WHERE
    date = '2022-06-01' AND
    NET.HOST(page) = NET.HOST(url) AND
    cert_issuer IS NOT NULL
```

Reply:
`NET.HOST(page) = NET.HOST(url)` will be true for all 1P requests per page. Could you use `WHERE firstHtml` instead to only have one request per page?

Reply:
Ah yes, you're right! I'll change it to `firstHtml`.